website/src/components/Layout/Header/styles.css

```diff
@@ -32,4 +35,10 @@ text-align: center;
   nav li.pipe{
     display: inline-block !important;
   }
+}
+
+.dropdown-menu{
+  background-color: #f8f9fa !important;
+  padding: 0.5rem 4.5em !important;
+  margin-left: -4em;
 }
```

Comment:
The github PR shows you're missing a newline at the end of your file. You have that in other files. Always add an empty newline at file ends (it helps with bash tools).